pkg/handlers/authorization/handler.go

```diff
@@ -50,58 +52,83 @@ func (auth *authorizationHandler) Name() string {
 
 //Process the request for authorization. The handler first attempts to get userinfo using bearer token
 //and falls back to the certificate subject or fails
-func (auth *authorizationHandler) Process(req *http.Request, context *handlers.RequestContext) (*http.Request, error) {
+func (auth *authorizationHandler) Process(req *http.Request) (*http.Request, error) {
```

Comment:
I still don't understand the reason for returning the HTTP request at all... Can't you just remove this and only return error?